APIDesign/README.md

```diff
@@ -199,6 +199,7 @@ All responses will include the following headers:
  * `Etag`: Use to specify the version of the resources as a UUID.
  * `Last-Modified`: Use to specify the date of the resource. See `If-Modified-Since`
 	 and `If-Unmodified-Since` above.
+ * `Surrogate-Key`: [a proprietary fastly feature](http://www.fastly.com/blog/surrogate-keys-part-1/)
```

Comment:
I think it would be helpful to add a few words about what this does and what it signifies. Readers can follow the link, but it would be nice if they didn't have to.